exercises/luhn/src/test/java/LuhnValidatorTest.java

```diff
@@ -19,12 +19,24 @@ public void testThatTheStringConsistingOfASingleZeroIsInvalid() {
 
     @Ignore
     @Test
-    public void testThatAValidCanadianSocialInsuranceNumberIsIdentifiedAsValid() {
+    public void testThatASimpleValidCanadianSocialInsuranceNumberIsIdentifiedAsValid() {
```

Comment:
Not that it matters very much, but I don't think this is a valid canadian SIN as it only has two digits. In the canonical data this test is called "simple valid sin" so maybe change this test name to `testThatASimpleValidSocialInsuranceNumberIsIdentifiedAsValid()`?